APE1.rst

```diff
@@ -139,6 +139,28 @@ An APE's status can
   rationales, so that new community members can understand why a decision was
   made.
 
+Modifications to an APE
+.......................
+
+**Addendum: July 22, 2020**
+
+An APE can be modified, with one person acting as "champion" for the
+APE update. Similar to the APE development procedure, the update
+champion should discuss their desired changes with the astropy
+community (e.g., through the astropy-dev mailing list).
+
+Following a discussion on astropy-dev, the APE update shall be
+submitted as a Pull Request. The submission of the Pull Request should
+be announced formally on the astropy-dev list before they are
+accepted. If not already listed, the update champion can add
+themselves to the author list for the APE being updated. When
```

Comment:
I think it's better to keep the metadata along with the addendum.  E.g. keeping the date of its addition, and the authors at the addendum header level